src/Backups/WithRetries.h

```diff
@@ -15,20 +17,11 @@ class WithRetries
 {
 public:
     using FaultyKeeper = Coordination::ZooKeeperWithFaultInjection::Ptr;
-    using RenewerCallback = std::function<void(FaultyKeeper &)>;
 
-    struct KeeperSettings
```

Comment:
This structure `WithRetries::KeeperSettings` was moved to `BackupKeeperSettings`.